tests/system/fim/scenarios/207_use_of_report_changes/.gitignore

```diff
@@ -0,0 +1 @@
+agents_outputs
```

Comment:
I just noticed the scenario folder, is too long and doesn't describe the trigger, the syntax na me is `id_description_trigger`, please consider changing it to `207_report_changes_frequency`